aws-resources/src/main/java/io/opentelemetry/contrib/aws/resource/IncubatingAttributes.java

```diff
@@ -62,6 +62,8 @@ private CloudProviderValues() {}
   public static final AttributeKey<String> K8S_CLUSTER_NAME =
       AttributeKey.stringKey("k8s.cluster.name");
 
+  public static final AttributeKey<String> AWS_ECS_CLUSTER_ARN =
+      AttributeKey.stringKey("aws.ecs.cluster.arn");
```

Comment:
Probably out of scope of this PR. But I don't see `aws.ecs.task.id` being populated while it is mentioned in the [spec](https://opentelemetry.io/docs/specs/semconv/attributes-registry/aws/#aws-ecs-attributes). Any plans on adding that too for being fully spec compliant?

Reply:
`aws.ecs.task.id` is recently added, and that's probably why it isn't implemented.

Reply:
We can do that in another PR, but I'll keep this clean and short.